crates/net/network/src/eth_requests.rs

```diff
@@ -83,7 +83,7 @@ where
         let mut block: BlockHashOrNumber = match start_block {
             BlockHashOrNumber::Hash(start) => start.into(),
             BlockHashOrNumber::Number(num) => {
-                if let Some(hash) = self.client.block_hash(num.into()).unwrap_or_default() {
+                if let Some(hash) = self.client.block_hash(U256::from(num)).unwrap_or_default() {
```

Comment:
there are a lot of `.into -> U256::from` changes. why is this now necessary?

Reply:
@recmo any odds this is fixable by making your method be called `uint_from` or `from_uint`?

https://github.com/recmo/uint/blob/main/src/from.rs#L101

i dont mind this change too much otherwise